### PR TITLE
Fix the bad mailers preview path

### PR DESCRIPTION
**Pull request by scharrier**

### CODE REVIEW

File: lib/roll/app_builder.rb [hunk 1]
```diff
@@ -79,11 +79,11 @@ def configure_generators
     end
 
     def configure_mailers_preview_path
-      config = <<-RUBY
+      config = <<-'RUBY'
```
Comment (scharrier) [inline lib/roll/app_builder.rb]: Adding ' disable interpolation, which is needed here (`Rails.root` shouldn't be interpolated before app execution)

File: lib/roll/app_builder.rb [hunk 2]
```diff
@@ -79,11 +79,11 @@ def configure_generators
...
 
 
   # Specific mailers path
-  config.action_mailer.preview_path = Rails.root + '/spec/mailers/previews'
```
Comment (scharrier) [inline lib/roll/app_builder.rb]: `Rails.root` isn't a string here. If we want to write it like this, we need to use `Rails.root.to_s`, which is just :poop:, isn't it ?

File: lib/roll/app_builder.rb [hunk 3]
```diff
@@ -79,11 +79,11 @@ def configure_generators
...
+  config.action_mailer.preview_path = "#{Rails.root}/spec/mailers/previews"
```
Comment (mehlah) [inline lib/roll/app_builder.rb]: Could be `Rails.root.join('spec/mailers/previews')`.
We won't need to disable interpolation for heredoc syntax.

Comment (scharrier) [inline lib/roll/app_builder.rb]: Thx !

**Pull request merged**